### PR TITLE
Chapter 13 thymeleaf

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/thymeleaf-views/_index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+weight: 13
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
```
Comment (gildedgardenia) [inline content/thymeleaf-views/_index.md:14]: Please add objectives and glossary terms to this page.

File: content/thymeleaf-views/reading/using-templates/_index.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+weight: 3
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+<!-- TODO: Update when conditionals-template is created -->
```
Comment (gildedgardenia) [inline content/thymeleaf-views/reading/using-templates/_index.md]: Please update this TODO item!

**Review by gildedgardenia — APPROVED** [review]
Two things to resolve in addition to the comments made here:
1. Please remember to add answers in comments with concept checks for easy upload to Canvas!
2. Where is the studio starter code? Is it in GitHub?

File: content/thymeleaf-views/_index.md (hunk 1 above)
Comment (jwoolbright23) [reply content/thymeleaf-views/_index.md:14]: objectives and glossary terms added.

**Comment by jwoolbright23** [comment]
> Two things to resolve in addition to the comments made here:
> 
>     1. Please remember to add answers in comments with concept checks for easy upload to Canvas!
> 
>     2. Where is the studio starter code? Is it in GitHub?

Studio starter code located here: https://github.com/LaunchCodeEducation/Java17-SpaDay. Also updated all check your understandings with solutions.

File: content/thymeleaf-views/reading/using-templates/_index.md (hunk 2 above)
Comment (jwoolbright23) [reply content/thymeleaf-views/reading/using-templates/_index.md]: Completed.

**Pull request merged**